### ISSUE TITLE
Add options to define the default state of LinkControl options

### ISSUE THREAD

**Issue by liviopv** [WordPress/gutenberg#57417]
## What problem does this address?
SEO and accessibility concerns aside, users should be allowed to define the default LinkControl state to match their workflows. If a site owner has reasons to set all links to "Open in a new tab", WordPress should give them the option to set that by default for all links, instead of manually toggling it for all links, losing precious time in managerial work.

## What is your proposed solution?
Add options on Global Styles or Settings > Writing allowing users to define the defaults to:

- Open in a new tab
- Mark as nofollow

**Comment by skorasaurus** [comment]
Regarding opening in a new tab, there is an issue for this at https://github.com/WordPress/gutenberg/issues/54824 and https://github.com/WordPress/gutenberg/issues/55395

**Comment by Mamaduka** [comment]
@skorasaurus, should we close this as duplicate?

**Comment by skorasaurus** [comment]
> @skorasaurus, should we close this as duplicate?

I don't think there's an issue for 'mark as nofollow' as default; both of the above issues are for opening in a new tab (I closed one of them just now).

**Comment by dhananjaykuber** [comment]
Add options on Global Styles or Settings > Writing allowing users to define the defaults to:

Open in a new tab
Mark as nofollow


For above we need to make changes in core instead of Gutenberg?
